earthfile2llb/converter.go

```diff
@@ -1098,7 +1098,7 @@ func (c *Converter) SaveImage(ctx context.Context, imageNames []string, pushImag
 			c.mts.Final.SaveImages = append(c.mts.Final.SaveImages, si)
 		}
 
-		if pushImages && imageName != "" && c.opt.UseInlineCache {
+		if hasPushFlag && imageName != "" && c.opt.UseInlineCache {
```

Comment:
These are just renames in this file, right?

Reply:
correct -- just renaming the function arg to make it more clear that it represents a `SAVE IMAGE` vs `SAVE IMAGE --push`.